python/kat/harness.py

```diff
@@ -1616,6 +1616,15 @@ def _setup_k8s(self, selected):
             ):
                 raise RuntimeError("Failed applying CRDs")
 
+            print("waiting for emissary-apiext server to become available")
+            if os.system(
```

Comment:
there seems to be a utility function `ShellCommand.run_with_retry` already. Why not use that to keep consistency?

Reply:
It is consistent with what is there in that file. If you look at that file we call `os.system` many times. Even a few lines below that. The `ShellCommand.run_with_retry` is currently only used in scenarios where we want a retry mechanism. 

In this scenario, I don't think we need a retry mechanism because I kept the existing applying mapping retry. So, between the startup probe and the mapping apply retry if it still errors at that point then more than likely something is actually wrong.